java/ant.debugger/src/org/netbeans/modules/ant/debugger/AntDebugger.java

```diff
@@ -306,9 +306,8 @@ private void stopHere(AntEvent event) {
         updateUI();
         currentFile = event.getScriptLocation();
         // update variable values
-        Set properties = event.getPropertyNames ();
-        variables = (String[]) properties.toArray 
-            (new String [properties.size ()]);
+        Set<String> properties = event.getPropertyNames();
+        variables = (String[])properties.toArray(new String [properties.size()]);
```

Comment:
Redundant cast after change
```suggestion
        variables = properties.toArray(new String [properties.size()]);
```

Reply:
Fixed..